CTP/conf/shell.conf

```diff
@@ -22,62 +22,41 @@
 #WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE 
 #USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# To configure host ip, port, user, password for instance1 test node to use ssh protocol
+# Define test instances for shell - the host ip, password, port and user for ssh connection.
+# the prefix and suffix will be used as keywords for CTP configuration, and the second section - 'instance1' can be changed, and
+# it will also be logged into file as reference information. For multiple sets of test node environment, the instance names must be different
 env.instance1.ssh.host=192.168.1.100
 env.instance1.ssh.port=22
 env.instance1.ssh.user=<user>
 env.instance1.ssh.pwd=<pwd>
 
-# The cubrid_port_id on instance1 test node
+# Define TCP port id for the CUBRID programs for instance1 to avoid port conflicts
 env.instance1.cubrid.cubrid_port_id=1569
 
-# The %query_editor port on instance1 test node
+# Define %query_editor port for instance1 test node to avoid port conflicts
 env.instance1.broker1.BROKER_PORT=30093
 
-# The %BROKER1 port on instance1 test node
+# Define the %BROKER1 port for instance1 test node to avoid port conflicts
 env.instance1.broker2.BROKER_PORT=30094
 
-# To configure the second test node
+# Define the second test node
 env.instance2.ssh.host=192.168.1.101
 env.instance2.ssh.port=22
 env.instance2.ssh.user=<user>
 env.instance2.ssh.pwd=<pwd>
-env.instance2.cubrid.cubrid_port_id=1569
-env.instance2.broker1.BROKER_PORT=30093
-env.instance2.broker2.BROKER_PORT=30094
+env.instance2.cubrid.cubrid_port_id=1570
+env.instance2.broker1.BROKER_PORT=30095
+env.instance2.broker2.BROKER_PORT=30096
 
-# If execute continue testing when testing is interrupted
-main.mode.continue=false
-
-# Url of build which you will use to test
+# Define the URL of build which will be used to test. If this parameter is not set or commented out, CTP will execute testing without build installation
 main.testbuild.url=http://127.0.0.1/REPO_ROOT/store_02/10.1.0.6876-f9026f8/drop/CUBRID-10.1.0.6876-f9026f8-Linux.x86_64.sh
 
-# The root path of test case, it should be checked out in each test node
+# Define the path of test cases used for testing, it should be checked out on test node, and CTP will get the total case list from the first test node.
```

Comment:
It's better to remove "and CTP will get the total case list from the first test node"